app/shared/layouts/PageWrapper.js

```diff
@@ -9,43 +8,34 @@ import { NavigationProvider } from '@shared/layouts/navigation/NavigationProvide
 import { UserProvider } from '@shared/user/UserProvider';
 import { CampaignStatusProvider } from '@shared/user/CampaignStatusProvider';
 import { CampaignProvider } from '@shared/user/CampaignProvider';
-import { InsecureCookieClearer } from '@shared/layouts/InsecureCookieClearer';
 import { ImpersonateUserProvider } from '@shared/user/ImpersonateUserProvider';
 
-const isProductTourRequest = () => {
-  const headersList = headers();
-  const activePath = headersList.get('x-pathname');
-  return activePath === '/product-tour';
-};
-
 const PageWrapper = ({ children, hideFooter }) => (
-  <InsecureCookieClearer>
-    <UserProvider>
-      <ImpersonateUserProvider>
-        <CampaignProvider>
-          <CampaignStatusProvider>
-            <NavigationProvider>
-              <div className="overflow-x-hidden">
-                <JsonLdSchema />
-                {!isProductTourRequest() && <Nav />}
```

Comment:
We don't need this anymore for the product tour iframe? 🤔 

Well.. doesn't look like this is working as intended any, so 🤷 

<img width="1728" alt="image" src="https://github.com/user-attachments/assets/bc4fe084-ab6e-4c02-b700-89d79182e604" />